talentsearch/resources/js/talentSearchConstants.ts

```diff
@@ -0,0 +1,3 @@
+// eslint-disable-next-line import/prefer-default-export
+export const BASE_URL =
```

Comment:
I went with lower camelCase based on `adminConstants`. Also, [messages uses talentSearch](https://github.com/GCTC-NTGC/gc-digital-talent/blob/2f07a98ed65636581c113de2ecd1328072cdf296/talentsearch/resources/js/components/SearchDashboard.tsx#L11) as well for a key, but I did that as well 😄 . `talentsearchConstants` looks a bit weird if we are going with camelCase, but I can be convinced otherwise.

Reply:
Yeah, now that I think about it, talent and search are definitely two different words, so the `talentSearch` camelCasing makes way more sense 🤔